compiler/codegen/symbol.cpp

```diff
@@ -2335,27 +2477,6 @@ GenRet FnSymbol::codegenPYXType() {
   return ret;
 }
 
-std::string FnSymbol::getPythonArrayReturnStmts() {
```

Comment:
Should this have stayed a function?

Reply:
It was only called once so I just moved its body into the code for array returns so that one could see the whole picture.